utility/meerkat_status.py

```diff
@@ -137,18 +137,14 @@ def get_rsc_sensors(ant, band):
         lna_h = rsc_values.pop("rsc_rx{}_mmic_enable.mmic1".format(band))
```

Comment:
you have repeated terms for `rsc_values["lnas"]`, `amp2_h`, 'amp2_v' and `rsc_values["amp2"]` on both conditions for setting `lna_h` and `lna_v`.

I would change this to:
```py
if band == "s":
    lna_h = rsc_values.pop("rsc_rx{}_mmic_enable.mmic1".format(band))
    lna_v = rsc_values.pop("rsc_rx{}_mmic_enable.mmic2".format(band))
else:
    lna_h = rsc_values.pop("rsc_rx{}_lna_h_power_enabled".format(band))
    lna_v = rsc_values.pop("rsc_rx{}_lna_v_power_enabled".format(band))
 
rsc_values["lnas"] = "ON" if lna_h and lna_v else "OFF"
amp2_h = rsc_values.pop("rsc_rx{}_amp2_h_power_enabled".format(band))
amp2_v = rsc_values.pop("rsc_rx{}_amp2_v_power_enabled".format(band))
rsc_values["amp2"] = "ON" if amp2_h and amp2_v else "OFF"   
```

Reply:
We don't have second stage amplifier sensors connected to the S-band packetizers(or at least it's not listed under the same sensor name). Had to trick the script to think that we do have.